yarn/src/main/scala/org/apache/spark/deploy/yarn/ExecutorDelegationTokenUpdater.scala

```diff
@@ -76,7 +76,10 @@ private[spark] class ExecutorDelegationTokenUpdater(
         SparkHadoopUtil.get.getTimeFromNowToRenewal(
           sparkConf, 0.8, UserGroupInformation.getCurrentUser.getCredentials)
       if (timeFromNowToRenewal <= 0) {
-        executorUpdaterRunnable.run()
+        // If the `timeFromNowToRenewal` equals 0, it's better to wait 1 minutes to schedule for
+        // avoid cycle calling and cause StackOverflow Exception.
```

Comment:
Oh I see, I assume someone manually changed the renew time for testing purposes?  I think perhaps the comment should be clarified saying if the renewal time is very small wait 1 minute to avoid calling over and over very quickly.  Or alternatively where we set spark.yarn.token.renewal.interval we could just do a min of 1 minute.